master/internal/experiment.go

```diff
@@ -116,19 +116,13 @@ func newExperiment(
 
 	var launchWarnings []command.LaunchWarning
 	if expModel.ID == 0 {
-		err := m.rm.ValidateResources(poolName, resources.SlotsPerTrial(), false)
-		if err != nil {
+		if _, launchWarnings, err = m.rm.ValidateResources(sproto.ValidateResourcesRequest{
+			ResourcePool: poolName,
+			Slots:        resources.SlotsPerTrial(),
+			IsSingleNode: false,
```

Comment:
can we set it from `resources.IsSingleNode()`?

previously this didn't matter because this check was duplicated in `core_experiment.go` above which you took out.

Reply:
done!